.mergify.yml

```diff
@@ -1,18 +1,28 @@
+queue_rules:
+  - name: default
+    conditions:
+      - "#approved-reviews-by>1"
+
 pull_request_rules:
-  - name: automerge to main with label automerge and branch protection passing
+  - name: Automatic merge on approval to the main branch
     conditions:
-      - "#approved-reviews-by>0"
+      - "#approved-reviews-by>=1"
       - base=main
-      - label=automerge
+      - label=A:automerge
     actions:
+      queue:
+        name: default
       merge:
         method: squash
-        strict: true
-  - name: backport patches to v4.2.x branch
+        commit_message_template: |
+          {{ title }} (#{{ number }})
+          {{ body }}
+
+  - name: Backport patches to the release/v7.0.x branch
     conditions:
       - base=main
-      - label=backport/4.2.x
+      - label=A:backport/v7.x.x
```

Comment:
```suggestion
      - label=A:backport/v7.0.x
```

Reply:
This really should be `A:backport/v7.x.x` along with the branch name being `release/v7.x.x`, but I digress lol.